client/signup/steps/site-information/index.jsx

```diff
@@ -53,6 +53,11 @@ class SiteInformation extends Component {
 
 	handleInputChange = ( { target: { name, value } } ) => {
 		this.setState( { [ name ]: value } );
+		if ( this.props.flowName === 'onboarding-dev' ) {
```

Comment:
I think it's better to create a specific version of this step for use in the onboarding flow, rather than having `this.props.flowName === 'onboarding-dev'` in the code. You can create new steps by reusing existing components.

Reply:
I should have left a code comment to indicate that this was just temporary. The `onboarding` flow is in live testing already and I don't want this to appear before it's fully baked. This is just to start getting things landed and testable in a flow we're not sending any traffic to.

Reply:
👍

Reply:
Oh and in this particular case I wanted to better profile if this needed debouncing/throttling and I just haven't gotten to it yet.